doc/api/api.md

```diff
@@ -224,6 +224,26 @@ Heketi is able to manage multiple GlusterFS clusters, each composed of a set of
 }
 ```
 
+### Set Cluster Flags
```

Comment:
This looks great. One small thing is that there's no corresponding link at the top of the document to this section.

Reply:
fixed, thanks